modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -33,10 +34,11 @@ spec:
       {
         "type": "cnv-bridge", <4>
         "bridge": "<bridge-interface>", <5>
-        "vlan": 1 <6>
+        "macspoofchk": true, <6>
+        "vlan": 1 <7>
       },
       {
-        "type": "cnv-tuning" <7>
+        "type": "cnv-tuning" <8>
```

Comment:
I have not seen this in ipsteam document. Is it very specific to downstream? plus how this and KMP work together. This also look for uniqueness of MAC and KMP does the same as well.

Reply:
Is this related to Edward's comment above?

Reply:
This shows up on kubevirt user-guide:

> Note: For some CNI plugins without native support for custom MAC addresses, there is a workaround, which is to use the tuning CNI plugin to adjust pod interface MAC address. This can be used as follows: